src/scripts/browserDeviceProfile.js

```diff
@@ -353,6 +353,23 @@ import browser from './browser';
         return 2;
     }
 
+/**
+ * Checks if the web engine supports secondary audio.
+ * @param {HTMLVideoElement} videoTestElement The video test element
+ * @returns {boolean} _true_ if the web engine supports secondary audio.
+ */
+export function canPlaySecondaryAudio(videoTestElement) {
+    // We rely on HTMLMediaElement.audioTracks
+    // It works in Chrome 79+ with "Experimental Web Platform features" enabled
+    // It doesn't work in Firefox 108 even with "media.track.enabled" enabled (it only sees the first audio track)
+    // It seems to work on Tizen 5.5+ (2020, Chrome 69+). See https://developer.tizen.org/forums/web-application-development/video-tag-not-work-audiotracks
+    // Assume webOS 5+ (2020, Chrome 68+) supports secondary audio like Tizen 5.5+
+    return !!videoTestElement.audioTracks
+        && !browser.firefox
+        && (browser.tizenVersion >= 5.5 || !browser.tizen)
+        && (browser.web0sVersion >= 5.0 || !browser.web0s || !window.NativeShell);
```

Comment:
What is the point of the NativeShell check here?

Reply:
This means `we are not in the app`.
Since we cannot reliably detect the webOS version in the built-in browser, we simply ignore version checking.

Reply:
We could do this instead: `browser.web0sVersion >= 5.0 || !browser.web0sVersion`.

What do you think of comments placed like this?
```js
export function canPlaySecondaryAudio(videoTestElement) {
    // We rely on HTMLMediaElement.audioTracks
    // It works in Chrome 79+ with "Experimental Web Platform features" enabled
    return !!videoTestElement.audioTracks
        // It doesn't work in Firefox 108 even with "media.track.enabled" enabled (it only sees the first audio track)
        && !browser.firefox
        // It seems to work on Tizen 5.5+ (2020, Chrome 69+). See https://developer.tizen.org/forums/web-application-development/video-tag-not-work-audiotracks
        && (browser.tizenVersion >= 5.5 || !browser.tizen)
        // Assume webOS 5+ (2020, Chrome 68+) supports secondary audio like Tizen 5.5+
        && (browser.web0sVersion >= 5.0 || !browser.web0sVersion);
}
```

Reply:
Oh breaking the comments down like that seems nice :+1:

Reply:
It looks like in your latest changes the NativeShell check is missing. Was that intentional?